tests/test_vec_normalize.py

```diff
@@ -150,17 +152,17 @@ def test_sync_vec_normalize():
 
     env.reset()
     # Initialize running mean
+    latest_reward = None
     for _ in range(100):
-        env.step([env.action_space.sample()])
+        _, latest_reward, _, _ = env.step([env.action_space.sample()])
+
+    # Check that unnormalized reward is same as original reward
+    original_latest_reward = env.get_original_reward()
+    assert np.allclose(original_latest_reward, env.unnormalize_reward(latest_reward))
 
     obs = env.reset()
     original_obs = env.get_original_obs()
-    dummy_rewards = np.random.rand(10)
-    # Normalization must be different
+    # Check that unnormalization works
+    assert np.allclose(original_obs, env.unnormalize_obs(obs))
+    # Normalization must be different (between different environments)
     assert not np.allclose(obs, eval_env.normalize_obs(original_obs))
-
-    sync_envs_normalization(env, eval_env)
```

Comment:
why did you remove this?

Reply:
Oh wups, accidentally went too far with removing the lines. Sorry!